Command/LoadDataFixturesDoctrineCommand.php

```diff
@@ -26,9 +26,9 @@ class LoadDataFixturesDoctrineCommand extends DoctrineCommand
     /** @var SymfonyFixturesLoader */
     private $fixturesLoader;
 
-    public function __construct(SymfonyFixturesLoader $fixturesLoader)
+    public function __construct(SymfonyFixturesLoader $fixturesLoader, ?ManagerRegistry $doctrine = null)
     {
-        parent::__construct();
+        parent::__construct($doctrine);
```

Comment:
Coming to think of it, we should throw a deprecation notice similar to the one in DoctrineBundle (see https://github.com/doctrine/DoctrineBundle/blob/eb068c210a8c735c087ac93f9754b2798aec00a8/Command/DoctrineCommand.php#L31..L37).

Reply:
Yeah, makes sense, I'll do it ASAP, thanks!

Reply:
Done.